<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="DateTimeGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>491</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>日期和时间</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>132</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="TimeLabel">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="DateTimeLabel">
        <property name="text">
         <string>日期时间</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTimeEdit" name="TimeEdit">
        <property name="displayFormat">
         <string>hh:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="DateTimeEdit">
        <property name="displayFormat">
         <string>yyyy/MM/dd hh:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="EditTime"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="EditDateAndTime"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="GetCurrentDateAndTime">
        <property name="text">
         <string>获取当前日期时间</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="DateLabel">
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="EditDate"/>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="DateEdit">
        <property name="displayFormat">
         <string>yyyy/MM/dd</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="SetDateAndTime">
        <property name="text">
         <string>设置日期时间</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="SetDate">
        <property name="text">
         <string>设置日期</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="SetTime">
        <property name="text">
         <string>设置时间</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="StringLabel">
        <property name="text">
         <string>字符串显示 </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="CalendarGroupBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>日历</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>351</width>
       <height>240</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="CalendarFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="SelectedCalendaDate">
        <property name="text">
         <string>选择的日期：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="SelectedDateLineEdit"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCalendarWidget" name="Calendar"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TimerGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>481</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>定时器</string>
    </property>
    <widget class="QProgressBar" name="TimerProgressBar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>411</width>
       <height>20</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>401</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QTimeEdit" name="TimePeriodEdit">
        <property name="date">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>z 毫秒</string>
        </property>
        <property name="time">
         <time>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="TimerStartButton">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="TimerStopButton">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TimePeriodLabel">
        <property name="text">
         <string>定时周期：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="SetTimePeriodButton">
        <property name="text">
         <string>设定周期</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLCDNumber" name="HourLcdNum"/>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="MinuteLcdNum"/>
      </item>
      <item row="2" column="2">
       <widget class="QLCDNumber" name="SecondLcdNum"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="TimePassLineEdit">
        <property name="text">
         <string>流逝时间：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
